### ISSUE TITLE
COVID-19 library in torchvision

### ISSUE THREAD

**Issue by AlexTS1980** [pytorch/vision#2877]
## 🚀 Feature
Library of models and dataset interfaces for COVID-19 models

## Motivation

There are quite a few models (feature extractors, mask segmentation, classifiers) for COVID-19, both in pytorch and tensorflow. They use different datasets, making it hard to scientists to compare results and extend their findings. It would be good to (re-implement) at least some models and dataset interfaces as a library in torchvision

## Pitch

Similar to the `models` and `datasets` in torchvision: `models` for the published models (see below), at least those that come with pretrained weights (e.g. COVIDNet-CT), and `datasets` for open-source labelled dataset interfaces: eg. CNCB-CT, UCSD, MedSeg, Zenodo, especially mask extraction.

## Alternatives

None that I know of

## Additional context

Some candidates include COVIDNet (x-rays), COVIDNet-CT (ct-scans), COVNet (ct-scans), JCS (ct-scans).

**Comment by vfdev-5** [comment]
@AlexTS1980 thanks for proposing this Feature Request. For instance, project maintainers/owners are working on defining the rules of what can be added as model or dataset. This detailed information will be exposed in CONTRIBUTING guide (https://github.com/pytorch/vision/pull/2663). cc @fmassa 

As alternatives, I can think of MONAI: https://github.com/Project-MONAI/MONAI 
nevertheless handling medical imagery is also a direction that torchvision would like to support...

**Comment by AlexTS1980** [comment]
> @AlexTS1980 thanks for proposing this Feature Request. For instance, project maintainers/owners are working on defining the rules of what can be added as model or dataset. This detailed information will be exposed in CONTRIBUTING guide (#2663). cc @fmassa
> 
> As alternatives, I can think of MONAI: https://github.com/Project-MONAI/MONAI
> nevertheless handling medical imagery is also a direction that torchvision would like to support...

What I mean is for example take [this ](https://github.com/UCSD-AI4H/COVID-CT/blob/master/baseline%20methods/DenseNet169/DenseNet_predict.py) or [this](https://github.com/haydengunraj/COVIDNet-CT/blob/master/run_covidnet_ct.py) model and implement them as a part of torchvision library, so that they could be instantiated like VGG or ResNet, e.g. through `covid_models` package, with pretrained weights, etc. Same with the datasets.

**Comment by oke-aditya** [comment]
I guess an easy way cold be to publish weights to `torch hub`. People can simply instantiatie `alexnet` or `vgg` from torchvision and load the weights using `torch.hub.load()`.

**Comment by dvolgyes** [comment]
Alexnet, VGG are proven architectures, and e.g. VGG is very frequently used in transfer learning.
There are hundreds of new networks, but as long as they don't provide a clear reusable architecture,
i don't see any point to incorporate it into a core library. Torch hub is perfect for this.
As these covid networks trained on very specific dataset, often not even too diverse dataset,
i don't see that they could be generic enough for an inclusion. 

What would be the advantage of having these networks in torchvision, instead of torch.hub? 
The only point i see is publicity, but no offense, i haven't read anything in these models which would
raised them clearly above average.
